ArduCopter/GCS_Mavlink.cpp

```diff
@@ -570,6 +570,48 @@ bool GCS_MAVLINK_Copter::set_home(const Location& loc, bool _lock) {
     return copter.set_home(loc, _lock);
 }
 
+MAV_RESULT GCS_MAVLINK_Copter::handle_command_int_do_reposition(const mavlink_command_int_t &packet) {
+    const bool change_modes = ((int32_t)packet.param2 & MAV_DO_REPOSITION_FLAGS_CHANGE_MODE) == MAV_DO_REPOSITION_FLAGS_CHANGE_MODE;
+    if (!copter.flightmode->in_guided_mode() && !change_modes) {
+        return MAV_RESULT_DENIED;
+    }
+
+    // sanity check location
+    if (!check_latlng(packet.x, packet.y)) {
+        return MAV_RESULT_DENIED;
+    }
+
+    Location requested_position {};
```

Comment:
normally in Copter and Rover we use the word "position" to mean an offset from the ekf origin.  We use location for real locations.  It's perhaps not a hard blocker but we would probably call this variable something like "requested_loc".

Reply:
Sure, I liked the symmetry with plane, but not going to raise a stink over it :)